### ISSUE TITLE
Sphinx cannot parse all links from build docs

### ISSUE THREAD

**Issue by JulianGro** [vircadia/vircadia-docs-sphinx#110]
The build docs contain links which are not parseable by Sphinx.
Example:
```
Follow the [build guide](BUILD.md) to figure out how to build Vircadia for your platform.
```
Would need to be:
```
Follow a [build guide](../BUILD_GENERAL) to figure out how to build Vircadia for your platform.
```

**Comment by digisomni** [comment]
I can just rename the main build file to be the correct name, then she should work!

**Comment by JulianGro** [comment]
If you were to rename the file and put it into the same folder it would still not work because we are specifically telling it to look for a `.md` file. Not only would that throw a warning, but it would link to a `.md` in the end when it is already a `.html` file.

**Comment by digisomni** [comment]
The other links work so I think that will work too.

**Comment by JulianGro** [comment]
That is true actually. Linking to the source files extension works. Only linking to the destination throws a warning. So you are right, renaming the main build file should fix it.

**Comment by JulianGro** [comment]
Partially fixed by #115
